icubam/www/handlers/db.py

```diff
@@ -87,13 +87,7 @@ def get(self, collection):
       get_fn = functools.partial(get_fn, max_date=max_ts)
       data = store.to_pandas(get_fn(), max_depth=1)
       if collection == 'all_bedcounts' and should_preprocess:
-        # this cached_data dict is just a way to tell predicu not to load the
-        # data from ICUBAM and use this already loaded data instead
-        cached_data = {'raw_icubam': data}
-        data = predicu.data.load_bedcounts(
-          cached_data=cached_data,
-          preprocess=True,
-        )
+        data = preprocess_bedcounts(data)
```

Comment:
Separating processors and loaders avoids doing awkward things with cached_data.

Reply:
👍